acceptance/testdata/secret/secret-org.txtar

```diff
@@ -0,0 +1,85 @@
+# Setup environment variables used for testscript
+env REPO=$SCRIPT_NAME-$RANDOM_STRING
+env2upper SECRET_NAME=$SCRIPT_NAME_$RANDOM_STRING
```

Comment:
> This isn't expanding like you'd expect because `os.Expand` is being given `SCRIPT_NAME_` and `RANDOM_STRING`.
> 
> You'd need to do:
> 
> ```
> env2upper SECRET_NAME=${SCRIPT_NAME}_${RANDOM_STRING}
> ```

> To be honest, it might be worth a follow up PR where we change all env vars to expand with {} and add a guideline.

Makes sense and aligns with common Shell scripting practices